src/runtime_src/core/common/query_requests.h

```diff
@@ -3105,6 +3106,21 @@ struct cu_read_range : request
   to_range(const std::string& str);
 };
 
+struct can_ps_kernel : request
```

Comment:
A comment preceding this struct would be helpful.
We have not been good at commenting the query structs, but let's at least do it for new ones.